scripts/backup.pl

```diff
@@ -219,14 +219,24 @@ sub Main {
 }
 
 # create new backup directory
+
+$BackupDir = abs_path($BackupDir);
+
 my $Home = $Kernel::OM->Get('Kernel::Config')->Get('Home');
 
+if( $BackupDir =~ /^$Home/ ) {
```

Comment:
Here I would do a more general check using https://metacpan.org/pod/Path::Class::Dir#$boolean-=-$dir-%3Econtains($other). But this can only be added in rel-10_1 as `Path::Class` is only required since OTOBO 10.1.1.